router/failed-events-manager.go

```diff
@@ -55,7 +56,7 @@ func (*FailedEventsManagerT) SaveFailedRecordIDs(taskRunIDFailedEventsMap map[st
 	}
 
 	for taskRunID, failedEvents := range taskRunIDFailedEventsMap {
-		table := fmt.Sprintf(`%s_%s`, failedKeysTablePrefix, taskRunID)
+		table := `"` + strings.ReplaceAll(fmt.Sprintf(`%s_%s`, failedKeysTablePrefix, taskRunID), `"`, `""`) + `"`
```

Comment:
I know this is probably a temporary measure. It would make more sense to perform this in `failedEventsHandler()` where both `SaveFailedRecordIDs()` and `FetchFailedRecordIDs()` are called.

Reply:
@chrikar The goal of this pr is to evade a security risk with respect to /v1/failed-events endpoint. `SaveFailedRecordIDs` function is not exposed outside. So, no risk.

Reply:
the HTTP handler should not know how/where failed events are stored, let alone that jobRunId needs to be escaped as if it was a postgresql identifier in the first place :)

Reply:
Oh! Apologies. I misread the changes in this pr. I thought the escaping is done on exposed functions `FetchFailedRecordIDs` & `DropFailedRecordIDs` and felt it is not necessary to modify `SaveFailedRecordIDs`.